buildozer/targets/android.py

```diff
@@ -193,7 +193,10 @@ def _get_available_permissions(self):
     def _set_win32_java_home(self):
         if 'JAVA_HOME' in self.buildozer.environ:
             return
-        import _winreg
+        if platform in ('win32'):
```

Comment:
This won't work, because `('win32')` isn't a tuple. There's no reason to not just use == here anyway.